src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml

```diff
@@ -35,16 +35,22 @@
                 </div>
             </div>
 
-            <h2 class="govuk-heading-m">What happens next</h2>
-
-            <p class="govuk-body"> We'll send the apprentice's certificate to the address you entered. You can <a class="govuk-link" asp-controller="CertificateHistory" asp-action="Index">check the status of the certificate</a> in the 'Completed assessments' section of your account.</p>
-
+            @if (Model.SelectedGrade != CertificateGrade.Fail)
+            {
+                <h2 class="govuk-heading-m">What happens next</h2>
+                <p class="govuk-body">
+                    We'll send the apprentice's certificate to the address you entered. You can <a class="govuk-link" asp-controller="CertificateHistory" asp-action="Index">check the status of the certificate</a> in the 'Completed assessments' section of your account.
+                </p>
+            }
             <h2 class="govuk-heading-m">@Localizer["DoNextHeader"]</h2>
             <ul class="govuk-list">
                 <li><a class="govuk-link" asp-controller="Search" asp-action="Index">@Localizer["DoNextSearch"]</a></li>
                 <li><a class="govuk-link" href="https://www.smartsurvey.co.uk/s/certification-transaction-complete/" target="_blank">@Localizer["DoNextFeedback"]</a></li>
-                <li>or</li>
-                <li><a class="govuk-link" asp-controller="Dashboard" asp-action="Index">Return to account home</a></li>
+                @if (Model.SelectedGrade != CertificateGrade.Fail)
```

Comment:
I don't think AC4 required any changes - it was a test scenario to show that the link at the end applied to both the Fail and the Pass - so this is now broken